managers/views.py

```diff
@@ -1,8 +1,36 @@
+from django.db.models import Count
+from rest_framework import renderers
 from rest_framework import viewsets
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from users.models import CustomUser
 from managers.models import Project
 from managers.serializers import ProjectSerializer
 
 
 class ProjectViewSet(viewsets.ModelViewSet):
     queryset = Project.objects.all()
     serializer_class = ProjectSerializer
+
+
+class ProjectsList(APIView):
+    renderer_classes = [renderers.TemplateHTMLRenderer]
+    template_name = 'managers/projects_list.html'
+
+    def get_queryset(self):
+        return Project.objects.all().order_by('id')
+
+    def get(self, request):
+        projects_queryset = self.get_queryset()
```

Comment:
The whole logic of this `get` method is actually selecting queryset, so it should be moved to `get_queryset` method.

Reply:
Done